[024] Einheitliches Design | E.G.

iobosse03 opened the pull request.

iobosse03 committed:
[024] fixed the to do counter

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 committed:
[024] rock, paper and scissor buttons now get focus after being clicked

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 committed:
[024] changed the scoreboard to be more user friendly

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 committed:
[024] changed the outcome text to be more user friendly

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

Lucerrion committed:
[024] logo changed
024 Einheitliches Design https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 committed:
Merge branch '024' into 024-Einheitliches-Design

iobosse03 committed:
[024] teampage: changed the width of the accordions

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 committed:
[024] Teampage: the text is now aligned with the accordion

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 committed:
[024] To-do list: styled text area

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

Lucerrion committed:
[024] Design h1 margin

024 Einheitliches Design https://app.asana.com/0/1205072348884926/1205181150417402/f

Lucerrion committed:
Merge branch 'main' of github.com:SpaTeamAcademy/Team_A

Lucerrion committed:
Merge branch '024-Einheitliches-Design' of github.com:SpaTeamAcademy/Team_A

iobosse03 committed:
[024] Rock paper scissors: changed "choose" to "chose"

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

SpaTeamAcademy commented:
- **RPS:** Text ist immer noch in "You" und "I" Formulierungen, damit nicht klar, wer welches Symbol gewählt hat. Außerdem sollte die Win/Lose/Draw Message größer sein als der sonstige Text. Evtl. könnte statt dem Text "You choose rock.
I choose paper." die gewählten Emojis aufgezeigt werden, zB.
> Player ✊ - ✋ Computer 

- **ToDo List**: Counter bitte linksbündig. + Button und Eingabefeld nicht auf einer Höhe. Bitte mit dem Team abstimmen, ob das Design für alle passt. Das Styling sollte damit bereits möglichst abgeschlossen sein.

iobosse03 committed:
[024] Rock paper scissors: changed outcome message

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 committed:
[024] To-do list: aligned the add button with the text area

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 committed:
[024] Rock paper scissors: added a function that changes strings to emojis

024 Einheitliches Design (CSS) https://app.asana.com/0/1205072348884926/1205181150417402/f

iobosse03 commented:
Code Review: H.C., E.G.
